src/Jira/Api/Client/CurlClient.php

```diff
@@ -117,12 +120,12 @@ public function sendRequest(
 				curl_setopt($curl, CURLOPT_POSTFIELDS, json_encode($data));
 			}
 		}
-		elseif ( $method == 'PUT' ) {
-			curl_setopt($curl, CURLOPT_CUSTOMREQUEST, 'PUT');
+		elseif ( $method == 'PUT' || $method == 'DELETE' ) {
```

Comment:
I think it would have been nicer to rebase & land #76 instead of putting it in your own code. (Especially since the rebase was needed due to a code style change you did yourself)